README.md

```diff
@@ -15,9 +15,10 @@
 # 🔐Docker-OpenVPN
 Out of the box stateless openvpn server docker image which starts in just a few seconds and doesn't require presistent storage. To get it running,  just copy & paste the snippet below and follow instructions in your terminal:
 ```bash
-docker run --cap-add=NET_ADMIN \
--p 1194:1194/udp -p 80:8080/tcp \
+docker run --name openvpn --cap-add=NET_ADMIN \
+-d -p 1194:1194/udp -p 80:8080/tcp \
```

Comment:
I'd suggest not to use detached mode in this quick start snipped as we explicitly offer user to follow onscreen instructions.